TribalWars/src/Infrastructure/DependencyInjection.cs

```diff
@@ -20,6 +23,11 @@ public static class DependencyInjection
             services.AddIdentity<User, Role>()
                 .AddEntityFrameworkStores<ApplicationDbContext>();
 
+            services.AddScoped<IBuildingsQueueRepository, BuildingsQueueRepository>();
+            services.AddScoped<IBuildingsQueueService, BuildingsQueueService>();
```

Comment:
Considering we are using clean architecture as a project structure, implementation of services (that are not use e.q. identity) shouldn't be registered in Infrastructure project.